lib/chai/utils/inspect.js

```diff
@@ -230,6 +241,28 @@ function formatArray(ctx, value, recurseTimes, visibleKeys, keys) {
   return output;
 }
 
+function formatTypedArray(ctx, value, recurseTimes, visibleKeys, keys) {
+  var output = [];
+
+  // Truncating TypedArray if it has more than 1000 elements
+  var l = value.length > 1000 ? 5 : value.length;
```

Comment:
We have `chai.config.truncateTreshold` which could be used as a metric - but I suppose my point was less about the cutoff figure, but more that the cutoff makes a dramatic change. If it was `var l = value.length > 1000 ? 1000 : value.length;` it would make much more sense (although 1000 is maybe a bit too high still, perhaps a number like 20 or so?)